Makefile

```diff
@@ -183,3 +183,13 @@ xcode-ci:
 	xcodegen --spec Samples/watchOS-Swift/watchOS-Swift.yml
 	xcodegen --spec TestSamples/SwiftUITestSample/SwiftUITestSample.yml
 	xcodegen --spec TestSamples/SwiftUICrashTest/SwiftUICrashTest.yml
+
+.PHONY: check-package-diff update-package-diff
+# Diff will return an error if the files are different, so we need to ignore it with `|| true`.
+check-package-diff:
+	diff Package.swift Package@swift-5.9.swift > current_package_diff.patch || true
+	diff ./Utils/expected_package_diff.patch current_package_diff.patch
```

Comment:
`m`: I think we should add an error message explaining what the problem is. This will almost always fail when you modify one of the two files, I believe. So we could explain that hey you updated one file of these files or maybe two. Please ensure that you apply the same changes to both files. Then update the diff with the command below.

Othewise, it can be quite confusing what the problem is and you have to ask.

Reply:
Message added